### ISSUE TITLE
Charts (Feature Request)

### ISSUE THREAD

**Issue by SariSultan** [lppkarl/Trady#84]
Charts are very important missing feature.

**Comment by lppkarl** [comment]
@SariSultan yup, i know that but i really don't have time to implement the feature. It would be good if you could create a PR on that. Thanks.

**Comment by andyb1979** [comment]
I have a chart library (but its commercial) which I've integrated to Trady. See [scichart.com](https://www.scichart.com/wpf-chart-features). Screenshot below. I plan to write a blog post to talk about how I've done this at some point.

![SciChart WPF Technical Indicators with Trady](https://www.scichart.com/Downloads/Clients/SciChart-Trady-Indicators-Stock-Charts2.png)
